### DISCUSSION TITLE
khoa-hoc/version-control-va-github/chuong-1-gioi-thieu-ve-version-control/1-3-cac-khai-niem-trong-version-control

### DISCUSSION THREAD

**Discussion by giscus** [Khoa-Hoc-Du-Lieu/docs#17] [Comments]
# khoa-hoc/version-control-va-github/chuong-1-gioi-thieu-ve-version-control/1-3-cac-khai-niem-trong-version-control

https://www.khoahocdulieu.org/khoa-hoc/version-control-va-github/chuong-1-gioi-thieu-ve-version-control/1-3-cac-khai-niem-trong-version-control

**Comment by khangme** [comment]
_Test_ **comment**

**Reply by khangme** [reply]
> _Test_ **comment**

Test comment from GitHub Discussions
